benches/coprocessor_executors/stream_aggr/mod.rs

```diff
@@ -142,21 +168,30 @@ fn bench_stream_aggr_count_1_first_group_by_int_col_real_col_2_groups(
 }
 
 #[derive(Clone)]
-struct Input {
+struct Input<M>
+where
+    M: criterion::measurement::Measurement,
+{
     /// How many rows to aggregate
     src_rows: usize,
 
     /// The aggregate executor (batch / normal) to use
-    bencher: Box<dyn util::StreamAggrBencher>,
+    bencher: Box<dyn util::StreamAggrBencher<M>>,
 }
 
-impl std::fmt::Debug for Input {
+impl<M> std::fmt::Display for Input<M>
+where
+    M: criterion::measurement::Measurement,
+{
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         write!(f, "{}/rows={}", self.bencher.name(), self.src_rows)
     }
 }
 
-pub fn bench(c: &mut criterion::Criterion) {
+pub fn bench<M>(c: &mut criterion::Criterion<M>)
+where
+    M: criterion::measurement::Measurement + 'static,
```

Comment:
Why sometimes it needs `'static`?

Reply:
@breeswish Honestly speeking, I didn't analysis the lifetime carefully. But these `'static`s are necessary.